Allow assigning a spacer component between items in a rendered collection (#2137)
ViewComponent/view_component: docs/guide/collections.md

  end
end
```

> ## Spacer Components

Reviewer: ```suggestion
## Spacer components
```

Since 3.20.0
{: .label }

> You can also specify a component instance to be rendered between instances of the main component by using the :spacer_component option:

Reviewer: ```suggestion
Set `:spacer_component` as an instantiated component to render between instances:
```

```erb
<%= render(ProductComponent.with_collection(@products, spacer_component: SpacerComponent.new)) %>

> 

Reviewer: ```suggestion
```

```

> ViewComponent will render the SpacerComponent component between each pair of ProductComponent components.

Reviewer: ```suggestion
Which will render the SpacerComponent component between each pair of `ProductComponent`s.
```
